systemtest/tmt/tests/strimzi/main.fmf

```diff
@@ -0,0 +1,79 @@
+test:
+  ./test.sh
+duration: 2h
+environment:
+  DOCKER_ORG: "strimzi"
+  DOCKER_TAG: "test"
+  TEST_LOG_DIR: "systemtest/target/logs"
+  TESTS: ""
+  EXCLUDED_TEST_GROUPS: "loadbalancer"
+adjust:
+  - environment+:
+      EXCLUDED_TEST_GROUPS: "loadbalancer,arm64unsupported"
+    when: arch == aarch64, arm64
+
+/smoke:
+  summary: Run smoke strimzi test suite
+  component: [smoke]
+  duration: 20m
+  tier: 1
+  environment+:
+    TEST_PROFILE: all
+    TEST_GROUPS: smoke
+    CLUSTER_OPERATOR_INSTALLTYPE: bundle
+
+/upgrade:
+  summary: Run upgrade strimzi test suite
+  component: [strimzi, kafka, upgrade]
+  duration: 5h
+  tier: 2
+  environment+:
+    TEST_PROFILE: all
+    TEST_GROUPS: upgrade
+    CLUSTER_OPERATOR_INSTALLTYPE: bundle
+
+/regression-operators:
+  summary: Run regression strimzi test suite
+  component: [strimzi, kafka, regression, operators]
+  duration: 10h
+  tier: 2
+  environment+:
+    TEST_PROFILE: regression
+    TEST_GROUPS: regression
```

Comment:
You can use the profiles instead of combination of groups and test paths:

- component regression - `-Pcomponents`
- operators regression - `-Poperators`
- upgrade - `-Pupgrade`
- smoke - `-Psmoke`